examples/plot_simulate_beta.py

```diff
@@ -0,0 +1,132 @@
+"""
+===============================
+09. Simulate beta modulated ERP
+===============================
+
+This example demonstrates how event related potentials (ERP) are modulated
+by prestimulus beta events.
+"""
+
+# Authors: Nick Tolley <nicholas_tolley@brown.edu>
+
+###############################################################################
+# Let us import hnn_core
+import hnn_core
+from hnn_core import simulate_dipole, law_model
+from hnn_core.viz import plot_dipole
+
+###############################################################################
+# We begin by instantiating the network model described in Law et al. 2021.
+# The model can be instantiated in the same pattern as ``default_network``
+# using ``law_model(params)`` as in previous exmaples. Leaving the arguments
+# empty loads the default parameter set default.json.
+net = law_model()
+
+###############################################################################
+# To demonstrate sensory depression, we will add an ERP similar to
+# :ref:`evoked example <sphx_glr_auto_examples_plot_simulate_evoked.py>`.
+# but modified to reflect that parameters used in Law et al. 2021.
+# Specifically, we are considering the case where a tactile stimulus is
+# delivered at 150 ms. 25 ms later, the first input to sensory cortex arrives
+# as a proximal drive, followed by one distal and a final late proximal drive.
```

Comment:
I am reading the paper as I review the code. A couple of points:

- it would be helpful to list the drives in temporal order
- also perhaps explain what each drive represents. E.g., first proximal drive represents the thalamic input, then before adding the next drive add a text block explaining that the second drive represents feedback from SII region, and finally the third proximal is due to thalamocortical loop. 
- I know we haven't done this in the past, but we should start to think of having some examples that are "tutorial style" and others which are more "example style". I am guessing that the ones reproducing a paper will be more tutorial style
- it might also be helpful to mention *why* the parameters are different. It's because they have been tuned to match the two conditions in the data

Reply:
> also perhaps explain what each drive represents. E.g., first proximal drive represents the thalamic input, then before adding the next drive add a text block explaining that the second drive represents feedback from SII region, and finally the third proximal is due to thalamocortical loop.

Along with our discussion to flush out the examples, I think this would belong better in the `plot_simulate_evoked.py` example when we add more thorough dialogue. I can make a brief reference to it here, but explaining the origin of the ERP is not the main purpose of this paper/tutorial.